include/dsn/cpp/rpc_stream.h

```diff
@@ -97,6 +98,9 @@ namespace dsn
         {
             _last_write_next_committed = true;
             _last_write_next_total_size = 0;
+#ifdef DSN_USE_THRIFT_SERIALIZATION
```

Comment:
why we need to couple rpc_stream with thrift? we also have other encoder/decoders being developed like protoc and bond ...